api/endpoint-tests/endpoint-coverage.js

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env node
+
+const fs = require('fs');
+const readline = require('readline');
+
+const openAPI = require('../routes/openAPI');
+
+const allEndpoints = JSON.parse(
+  fs.readFileSync(`${__dirname}/endpoints-all.txt`)
+).reduce((acc, { path, method }) => {
+  const methodStatus = {
+    tested: false,
+    documented: false
+  };
+  if (acc[path]) {
+    acc[path][method] = methodStatus;
+  } else {
+    acc[path] = {
+      [method]: methodStatus
+    };
+  }
+
+  const openAPIpath = path.replace(/\/:(.+?)(\/|$)/, '/{$1}$2');
+  const oa = openAPI.paths[openAPIpath];
+  if (oa && oa[method] && oa[method].responses) {
+    acc[path][method].documented = Object.keys(oa[method].responses)
+      .map(r => +r)
+      .sort();
+  }
+
+  return acc;
+}, {});
+
+const hitsStream = fs.createReadStream(`${__dirname}/endpoints-hit.txt`);
+const hitsReader = readline.createInterface({ input: hitsStream });
+
+hitsReader.on('line', line => {
+  const { path, method, status } = JSON.parse(line);
+
+  if (!allEndpoints[path][method.toLowerCase()].tested) {
+    allEndpoints[path][method.toLowerCase()].tested = [];
+  }
+  if (!allEndpoints[path][method.toLowerCase()].tested.includes(status)) {
+    allEndpoints[path][method.toLowerCase()].tested.push(status);
+  }
+  allEndpoints[path][method.toLowerCase()].tested.sort();
+});
+
+hitsReader.on('close', () => {
+  const messages = [];
+  const methodMessages = [];
+
+  Object.keys(allEndpoints).forEach(path => {
+    messages.length = 0;
+
+    Object.keys(allEndpoints[path]).forEach(method => {
+      methodMessages.length = 0;
+      const tests = allEndpoints[path][method].tested;
+      const docs = allEndpoints[path][method].documented;
+
+      if (!tests) {
+        methodMessages.push(`    not tested`);
```

Comment:
is something missing here (`    not tested`)? if no, consider `\t` for consistent indention

Reply:
I'm going to leave it for now because my end goal is to output markdown and have it posted as a Github comment, but we'll see if I can get that far.  😉